### PR TITLE
Updated mod.py

### PR DESCRIPTION
**Pull request by fartcan6**
plz dont ban me from this org..

### CODE REVIEW

**Comment by fartcan6** [comment]
updated it!!!!

**Comment by JanCraft** [comment]
You should not be allowed access to the internet. How could you have enough brain power to pull request to a GitHub repository but not to understand how to properly name your commits and PRs and give them proper descriptions. It's inacceptable to name a PR "Updated X" without then providing a proper description of what you did. Moreover, your commit has a name that does not describe the changes applied to the file(s) you edited. On top of all of this, your code is not properly commented and does not indicate why*the code is doing what it is doing, this makes other contributors have a hard time understanding why that change was made. /srs /totally serios /furious

**Comment by fartcan6** [comment]
TL;DR

**Comment by asdfugil** [comment]
.

**Comment by fartcan6** [comment]
'm sure you've all heard about the recent allegations against me, and I knew I couldn't stay silent about this any longer. It's time to finally come clean and explain exactly what happened, and the steps that I'm taking to address the problem.

First of all, let me say that I am sorry. I know I fucked up, and I can't take back what I've done. I'm a human, and sometimes I make mistakes. That's not an excuse, but it is the truth. My heart goes out to the people that may or may not have been harmed by my actions. A number of people - I won't name names - have spoken out against what I've done, and they're right to do so. I deeply regret my actions, and while I don't expect them to forgive me, I do hope that the actions I've taken since to prevent this from happening again will help ease their suffering. I was in the wrong here, and they have every right to be angry at me. Are all of their accusations true? Who's to say. Many of the things they have allegedly claimed I have done may have been true, and many of them might not have been. It's important to consider the context that these words or actions might have taken place in, and withhold judgement until all the facts about this situation have become more clear. A lot of misinformation has been spreading about the things I did, and no one, not even me, can be entirely sure what exactly took place.

That being said, I deeply regret what happened, and take full responsibility for the consequences of whatever it was that I did. I had an opportunity to try and make things better, and it's clear that the choices I made were wrong, and only made things worse. I know that these words won't make everything right. Nothing I can do now will make this easier for the people who were affected by my actions, and I don't expect it to. All I can do now is be as honest as possible and make sure everyone understands why I did what I did. This whole experience has been extremely humbling, and has shown me how important it is to consider to consequences of carelessness for someone in my position. I have dedicated myself to ensuring that the things I have done will never happen again, in all likelyhood.

Above all else, I want everyone to know that my heart goes out to those who were involved in this, and that I am taking total responsibility for what I have done. I hope you will consider all the facts I have laid out here, and come to your own conclusions about this whole situation.

Finally, if anyone else feels the need to publicly apologize, or respond to accusations against them, I'd like to offer this thread as a space to do so. The public deserves to understand these situations from all sides, and I hope we can open a dialogue to make sure that these kinds of incidents don't happen again.

**Comment by mineek** [comment]
what the bananas is going on here

**Comment by JanCraft** [comment]
hi mineek

**Comment by fartcan6** [comment]
hi mineek!!
